java/shared/src/test/java/com/squareup/subzero/shared/SubzeroUtilsTest.java

```diff
@@ -526,4 +610,34 @@ private TxOutput testOutput(long amount, Destination destination) {
             .setIndex(2))
         .build();
   }
+
+  private ECKey newMockPubKey() {
+    return new ECKey();
+  }
+
+  private Signature createValidMockSignature(byte[] hash, ECKey pubkey) {
```

Comment:
naming nit: ECKey is a key pair (since it's being used to sign below), not a public key

naming nit: this is not a mock signature since nothing is being mocked. We're actually creating a real signature over the provided data with a real key. Maybe call it `createValidSignature()`?